packages/block-library/src/post-comments-form/index.php

```diff
@@ -28,6 +28,9 @@ function render_block_core_post_comments_form( $attributes, $content, $block ) {
 	$form               = ob_get_clean();
 	$wrapper_attributes = get_block_wrapper_attributes( array( 'class' => $classes ) );
 
+	// Enqueue the comment-reply script.
+	wp_enqueue_script( 'comment-reply' );
```

Comment:
Thanks for the update, I think the in_footer is by default to false (which is different from what you  had written previously). Not sure if that's an issue or not?

Reply:
In this case, it gets added to the footer automatically, I believe because enqueing happens on render so after the head 👍